teleport.lua

```diff
@@ -247,9 +256,11 @@ function do_teleport(pos, player)
 		dest.y = dest.y + 0.5
 
 		if telemosaic_digiline_switching ~= nil then
-			local chan = minetest.get_meta(pos):get_string("channel")
-			digilines.receptor_send(pos, digilines.rules.default, chan,
-				"telemosaic "..player:get_player_name().." "..hash_pos(pos))
+			local chan = meta:contains("channel") and meta:get_string("channel") or DEFAULT_CHANNEL
+			digilines.receptor_send(pos, digilines.rules.default, chan, {
+				player = player:get_player_name(),
+				hashpos = { origin = hash_pos(pos), target = dest_hash }
```

Comment:
minor thingh and could be done later if needed but would it be better to return hash_pos and dest_has as coordinate tables as those are easier to manipulate with lua controllers?
Or maybe add origin and target like this:
```lua
			digilines.receptor_send(pos, digilines.rules.default, chan, {
				player = player:get_player_name(),
				hashpos = { origin = hash_pos(pos), target = dest_hash },
				origin = pos,
				target = unhash_pos(dest_hash),
			})
```

Reply:
Done the suggested way (but move so that not recomputed the unhashed position)